async_fuse/src/main.rs

```diff
@@ -204,6 +209,18 @@ fn main() -> anyhow::Result<()> {
                         required argument, no default value",
                 ),
         )
+        .arg(
+            clap::Arg::with_name(CACHE_CAPACITY_ARG_NAME)
+                .short("c")
+                .long(CACHE_CAPACITY_ARG_NAME)
+                .value_name("CACHE_CAPACITY")
+                .takes_value(true)
+                .required(false)
+                .help(
+                    "Set the mount point of FUSE, \
```

Comment:
Looks like this is for Cache capacity, right?

Reply:
Yes, I fixed it in later commit, not included in this PR. Let me fix it here. Thanks.